chromium_src/chrome/browser/ui/views/web_apps/web_app_views_utils.cc

```diff
@@ -0,0 +1,22 @@
+// Copyright (c) 2023 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// You can obtain one at https://mozilla.org/MPL/2.0/.
+
+#include "chrome/browser/ui/views/web_apps/web_app_views_utils.h"
+
+#define CreateOriginLabel CreateOriginLabel_ChromiumImpl
+#include "src/chrome/browser/ui/views/web_apps/web_app_views_utils.cc"
+#undef CreateOriginLabel
+
+namespace web_app {
+
+std::unique_ptr<views::Label> CreateOriginLabel(const url::Origin& origin,
```

Comment:
don't we already have an override for `FormatOriginForSecurityDisplay`? If not, it seems like that would be a better place to handle this more generally?

Reply:
Created follow up issue https://github.com/brave/brave-browser/issues/32950